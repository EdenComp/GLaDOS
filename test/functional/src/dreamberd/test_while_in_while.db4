int i = 0;
int j = 0;
int k = 0;

while (i < 3) {
    j = 0;
    while (j < 3) {
        k = 0;
        while (k < 3) {
            print(i+j+k+"\n");
            k = k + 1;
        }
        print(i+j+"\n");
        j = j + 1;
    }
    print(i+"\n");
    i = i + 1;
}
