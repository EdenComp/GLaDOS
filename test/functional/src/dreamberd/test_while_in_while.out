000
001
002
00
010
011
012
01
020
021
022
02
0
100
101
102
10
110
111
112
11
120
121
122
12
1
200
201
202
20
210
211
212
21
220
221
222
22
2
